common/main/en_CA.xml

```diff
@@ -1883,10 +1883,10 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 						<dateFormatItem id="MMMMW" count="other">↑↑↑</dateFormatItem>
 						<dateFormatItem id="ms">↑↑↑</dateFormatItem>
 						<dateFormatItem id="y">↑↑↑</dateFormatItem>
-						<dateFormatItem id="yM">↑↑↑</dateFormatItem>
-						<dateFormatItem id="yMd">↑↑↑</dateFormatItem>
+						<dateFormatItem id="yM">MM/y</dateFormatItem>
```

Comment:
We should always have the order: y then M then d

Would be good to have a test for this, whenever the pattern contains M or MM.

Reply:
i'm not quite following.  the modified `y`'s are:

```xml
						<dateFormatItem id="y">↑↑↑</dateFormatItem>
						<dateFormatItem id="yM">MM/y</dateFormatItem>
						<dateFormatItem id="yMd">y-MM-dd</dateFormatItem>
						<dateFormatItem id="yMd" alt="variant">d/M/y</dateFormatItem>
						<dateFormatItem id="yMEd">E, y-MM-dd</dateFormatItem>
						<dateFormatItem id="yMEd" alt="variant">E, d/M/y</dateFormatItem>
```

Which one is out of order?

Reply:
Mark, do you mean MM/y should be y/MM?

Reply:
ptal